app/assets/javascripts/rating.js

```diff
@@ -1,5 +1,5 @@
 $(function(){
-  function handleScroll(e) {
+  function handleScroll() {
```

Comment:
What has this change to do with this PR 🤔 ?

Reply:
I fixed it because the JShint said 'e' var is not usable. And JShint/Rubocop broked my build

Reply:
I see 💡 
Still, this is definitely already somehow fixed on master - the build is ✅  there. I'd suggest to undo this change and get this branch up to date with master.